docs/README.md

```diff
@@ -50,7 +49,8 @@ Now, you can use all blocks exported by the `store-form` app. See the full list
      "product-questions-and-answers"
    ]
  },
-```
+...
+}
```

Comment:
Substituir as linhas 40 até 53 por:

```
    ```json
    {
        "store.product": {
            "children": [
                "flex-layout.row#product-breadcrumb",
                "flex-layout.row#product-main"
                "flex-layout.row#form",
                "shelf.relatedProducts",
                "product-reviews",
                "product-questions-and-answers"
            ]
        },
        ...
    }
    ```
```